pom.xml

```diff
@@ -51,6 +51,7 @@
         <version.dozer>5.5.1</version.dozer>
         <version.slf4j>1.7.25</version.slf4j>
         <version.logback>1.2.3</version.logback>
+        <version.craftsman>1.0.3</version.craftsman>
```

Comment:
`craftsman` is the name of the group, not the actual dependency, since that is `dozer-jdk8-support`.
This property name can be misleading, try to rename it to something like `version.dozerjdk8`.